src/mapml.css

```diff
@@ -155,6 +151,43 @@
   text-decoration: revert;
 }
 
+
+/*
+ * Attribution.
+ */
+
+.leaflet-container .leaflet-control-attribution {
+  background-color: #fff;
+  border-radius: 1.5em;
+  color: currentColor;
```

Comment:
This may just be a personal preference but the button may fit the overall theme if it's dark on a light background rather than light on a dark background.

Reply:
I used dark on light initially, but felt better after trying light on dark. They're different variants of the SVG though. I can post screenshots of dark on light too, and we'll decide then?

Reply:
I'll take your word for it, it probably has better contrast being light on dark anyways.

Reply:
A compromise would be dark on light but with a dark (circular) stroke instead?

Reply:
Changed to dark on light in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/642/commits/c6420be80d66a2517a31bac9a9a79912dbe68309. I think it is a better default, and once we've enabled customization (using CSS Parts or Custom properties) the theme color can easily be changed by developers.

<img width="450" src="https://user-images.githubusercontent.com/26493779/148725260-5e0e29d5-ff4f-4a91-8f19-2707d1f59408.png">